scrapy_zyte_api/_params.py

```diff
@@ -591,12 +591,13 @@ def __init__(self, crawler, cookies_enabled=None):
 
     def parse(self, request):
         dont_merge_cookies = request.meta.get("dont_merge_cookies", False)
+        use_default_params = request.meta.get("zyte_api_default_params", True)
         cookies_enabled = self._cookies_enabled and not dont_merge_cookies
         params = _get_api_params(
             request,
-            default_params=self._default_params,
-            transparent_mode=self._transparent_mode,
-            automap_params=self._automap_params,
+            default_params=self._default_params if use_default_params else {},
+            transparent_mode=self._transparent_mode if use_default_params else False,
```

Comment:
Why is disabling transparent mode needed? Won't it be disabled anyways if `zyta_api` meta key is used?